src/panels/config/devices/device-detail/ha-device-entities-card.ts

```diff
@@ -92,15 +92,19 @@ export class HaDeviceEntitiesCard extends LitElement {
 
     return html`
       <ha-card outlined .header=${this.header}>
-        <div id="entities">
-          <mwc-list>
-            ${shownEntities.map((entry) =>
-              this.hass.states[entry.entity_id]
-                ? this._renderEntity(entry)
-                : this._renderEntry(entry)
-            )}
-          </mwc-list>
-        </div>
+        ${shownEntities.length
+          ? html`
+              <div id="entities">
+                <mwc-list>
+                  ${shownEntities.map((entry) =>
+                    this.hass.states[entry.entity_id]
+                      ? this._renderEntity(entry)
+                      : this._renderEntry(entry)
+                  )}
+                </mwc-list>
+              </div>
+            `
+          : ""}
```

Comment:
```suggestion
          : nothing}
```

you need to import `nothing` from lit